### DISCUSSION TITLE
Show all feeds?

### DISCUSSION THREAD

**Discussion by tao4mike** [FreshRSS/FreshRSS#7143] [Q&A] [answered]
Is there a specific URL that will show articles from all feeds? Specifically, I'm trying to get the URL needed to append a search query to, which will search all articles. Using /i/?search=test just seems to search the "Main Stream", and I don't have all feeds added to that obviously. I've done a cursory search, and nothing seems to point to the answer. Even AI is stumped. Thanks for any help!

**Comment by Frenzie** [comment]
Did you try toggling read? In URL terms, &state=3

**Reply by tao4mike** [reply]
Thanks for the reply. I'm not specifically looking for read/unread articles, but rather any articles across all feeds. That didn't seem to do the trick. I appreciate the suggestion!

**Reply by Alkarex** [reply]
> but rather any articles across all feeds

https://github.com/FreshRSS/FreshRSS/discussions/7143#discussioncomment-11667750

**Comment by Alkarex** [comment]
Not tried yet, but here is a quick draft https://github.com/FreshRSS/FreshRSS/pull/7144 with `&get=A`

**Comment by Alkarex** [comment, marked as answer]
Implemented in https://github.com/FreshRSS/FreshRSS/pull/7144

**Reply by tao4mike** [reply]
Alexadre. I am somewhat new to interacting with this community (github), so I hope it's appropriate to say thank you for getting this out so fast! I was away for the weekend, so I only had my phone to watch the progress, and I greatly appreciate it!
